extensions/arc/package.nls.json

```diff
@@ -20,6 +20,8 @@
 	"arc.data.controller.kube.cluster.context": "Cluster context",
 	"arc.data.controller.cluster.config.profile.title": "Choose the config profile",
 	"arc.data.controller.cluster.config.profile": "Config profile",
+	"arc.data.controller.cluster.config.profile.loading": "Loading config profiles",
+	"arc.data.controller.cluster.config.profile.loadingcompleted": "Done loading config profiles",
```

Comment:
can we use "loading config profiles completed", the one you have will make me think it is "Downloading confg profiles" :)